tests/integration/expected_test_results/api_v2_product_read/get-fields-all-knowledge-panels.json

```diff
@@ -1808,6 +1808,12 @@
                   "panel_element" : {
                      "panel_id" : "environment_card"
                   }
+               },
+               {
+                  "element_type" : "panel",
+                  "panel_element" : {
+                     "panel_id" : "contribution_card"
```

Comment:
We seem to be referencing a panel contribution_card that does not exist.

Reply:
contribution_card.tt.json